### PR TITLE
[ODS-3912] Sandbox Admin App support for PostgreSQL

### PR DESCRIPTION
**Pull request by jamessmckay**

### CODE REVIEW

File: Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs [hunk 1]
```diff
@@ -2,73 +2,37 @@
...
         }
 
         public static string SampleDatabase
         {
-            get { return DatabaseName(TemplateSampleDatabase); }
-        }
-
-        private static string DatabaseName(string databaseName)
-        {
-            return DatabaseNameTemplate.Contains("{0}")
```
Comment (tbmeador-dlp) [inline Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs:54]: doesn't this remove the configurability of the templated connection string?
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/03a24fe2ecc8fbab411efb8656395e4995781294/Application/EdFi.Ods.SandboxAdmin.Web/Web.Npgsql.config#L14-L16

Comment (jamessmckay) [reply Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs:54]: This was never used, that is why i removed it.

Comment (tbmeador-dlp) [reply Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs:54]: What do you mean? The `DatabaseNameTemplate` pulls the EdFi_Ods connection string from the web.config and  formats it with the populated template names. Our deployment sets it to `Database=EdFi_EFAStaging#{Octopus.Release.Channel.Name}_{0};` which allows it to differentiate sandboxes between different api versions.

Comment (jamessmckay) [reply Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs:54]: done

Comment (jamessmckay) [reply Application/EdFi.Admin.DataAccess/Utils/DatabaseNameBuilder.cs:54]: done

**Pull request merged**